I believe we've got everything... please review!

JerroldJV opened the pull request:
This includes fixes to bugs #41 #42 #43 #47 #48 #50 #51 #54 #56

JerroldJV committed:
Fixes to move validation

JerroldJV committed:
Just.... stuff

rsfriedman commented:
I might be doing something wrong but I encountered the following bug:

I make a suggestion as player 1. It displays "You have an open suggestion please wait". For the other players it displays it is not their turn and does display that the other person has made a suggestion. I'm unable to actually do anything at this point. Aren't we supposed to be able to suggest cards to the suggestion as the other players?

Wait..it showed up like several minutes later. That's a very slow delay...

Update: this seems to work fine with 2 players but the delay happens with 3....

JerroldJV committed:
Fixed bug with getNextPlayer

If the player who you are calling getNextPlayer on has lost, and the
removeLosingPlayers is set to true, this method would always return
None.  Fixed logic around

JerroldJV committed:
Removed debuggin print statements
